apps/testing-input-output/src/app/counter.component.cy.ts

```diff
@@ -1,11 +1,36 @@
+import { createOutputSpy } from 'cypress/angular';
 import { CounterComponent } from './counter.component';
 
 describe(CounterComponent.name, () => {
+  function increment() {
+    cy.contains('button', 'Increment').click();
+  }
+
+  function send() {
+    cy.contains('button', 'Send').click();
+  }
+
   it('using createOutputSpy', () => {
-    cy.mount(CounterComponent);
+    cy.mount(CounterComponent, {
+      componentProperties: {
+        send: createOutputSpy('sendSpy'),
+      },
+    });
+
+    increment();
+    send();
+
+    cy.get('@sendSpy').should('have.been.calledWith', 1);
   });
 
   it('using autoSpyOutputs', () => {
-    cy.mount(CounterComponent);
+    cy.mount(CounterComponent, {
+      autoSpyOutputs: true,
+    });
+
+    increment();
```

Comment:
nice. really like this function to create readable test.

I might call it. clickOnIncrementButton() to really know what this is doing. but naming has always been a dev battle 🤣

Reply:
Thanks - changed it to clickOnIncrement**Control** (inspired by [testing-library](https://github.com/testing-library/angular-testing-library/blob/main/apps/example-app/src/app/examples/02-input-output.spec.ts#L19)).